src/__tests__/questions.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Questions from '../Questions';
```

Comment:
You don't actually import the Questions component here but Questions.json instead. That's why the test doesn't work. You try to mount() an array of objects and not a function.
Import mechanism work as follows:
1) if file extension isn't provided JavaScript looks for .js
2) then .json (Here's your case)
3) and only then all the rest (including .jsx)
P.S. if your .json file was called starting with a lowercase letter (as long as it is not a component it's more correct semantically) you would not have gotten into such a problem.